services/api-server/src/simcore_service_api_server/models/schemas/jobs.py

```diff
@@ -306,3 +307,13 @@ class JobLog(BaseModel):
     node_id: NodeID | None
     log_level: LogLevelInt
     messages: list[LogMessageStr]
+
+    class Config(BaseConfig):
+        schema_extra: ClassVar[dict[str, Any]] = {
+            "example": {
+                "job_id": "145beae4-a3a8-4fde-adbb-4e8257c2c083",
+                "node_id": "3742215e-6756-48d2-8b73-4d043065309f",
+                "log_level": logging.DEBUG,
+                "messages": ["PROGRESS: 5/10"],
```

Comment:
minor: `"PROGRESS: 5/10"` looks like it actually has a format (it is not an arbitrary string but we expect the prefix PROGRESS in capital letters and such). So maybe this should be wrapped as some class with validation or whatever instead of hardcoding? But Im sure it's fine for now

Same for the UUIDs. Also i just notice that `job_id` would formally be `job_uuid` to be clear. But I am getting very pedantic this is all just style and coder-preference :)

Reply:
Very valid points. Concerning the message it can actually be any string. The most interesting logs for the user are the ones stating the progress, but it can really be any string. Yes, I agree we could/should change to `job_uuid` instead of `job_id` but that will have to be done throughout the api-server. So I will not handle that now. Thanks for the input.